syncr_backend/constants.py

```diff
@@ -0,0 +1,27 @@
+NODE_ID_BYTE_SIZE = 32
+DROP_ID_BYTE_SIZE = 64
+
+# Request structure indices for Tracker
+TYPE_INDEX = 0
```

Comment:
can you prepend these with `TRACKER_` so we know these are tracker values?

Although in the long run I probably want to make a data structure (class or typed tuple) that does this rather than relying on constants.